tools/lint/lib/checks/license.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from ..check import Check
+
+_MIN_YEAR = 2009
+_MAX_YEAR = 2030
+
+_LICENSE_PATTERN = re.compile(
+    r'\/\/ Copyright( \([cC]\))? (\w+) .+\. {1,2}All rights reserved\.[\r\n]{1,2}' +
```

Comment:
regexp is hard, but would this work with files containing multiple copyright lines?

wrt the comment on #851, I'm not even sure this is worth investigating.